main/webapp/modules/core/scripts/views/data-table/data-table-view.js

```diff
@@ -149,7 +149,7 @@ DataTableView.prototype._renderSortingControls = function(sortingControls) {
 
   $('<a href="javascript:{}"></a>')
   .addClass("action")
-  .text("Sort ")
+  .text($.i18n('core-views/first') + " ")
```

Comment:
This is a good start! However it's not enough for this string to be translatable - it also needs to say the right thing :)
At the moment this will show "First" instead of "Sort", which would be quite confusing. So the identifier for the translatable string needs to be different.

Reply:
ahhh yes, that is really an obvious mistake 🤦‍♂ Have made the necessary change.